setup.py

```diff
@@ -259,9 +260,117 @@ def run(self):
         remove_sprites_from_folder(self.buildfolder / "data" / "sprites" / "alttpr")
 
         self.create_manifest()
-
         if is_windows:
             # Inno setup stuff
+            # did not find a good source to automatically pull these
+            default_rom_file_names: typing.Dict[str, str] = {
+                "A Link to the Past": "Zelda no Densetsu - Kamigami no Triforce (Japan).sfc",
+                "Super Metroid": "Super Metroid (JU).sfc",
+                "Donkey Kong Country 3": "Donkey Kong Country 3 - Dixie Kong''s Double Trouble! (USA) (En,Fr).sfc",
+                "Super Mario World": "Super Mario World (USA).sfc",
+                "Secret of Evermore": "Secret of Evermore (USA).sfc",
+                "Lufia II Ancient Cave": "Lufia II - Rise of the Sinistrals (USA).sfc",
+            }
+
+            snes_md5s: typing.Dict[str, str] = {
+                "A Link to the Past": "03a63945398191337e896e5771f77173",
```

Comment:
isn't the hash available through AutoPatch? if not, this should be made available through that, instead of listing the hashes here?